src/prommis/uky/uky_flowsheet.py

```diff
@@ -1732,121 +1740,121 @@ def function(unit):
+        #     solver.solve(m.fs.solex_rougher_strip, tee=True)
+        #     # Unfix feed states
+        #     m.fs.solex_rougher_strip.mscontactor.organic_inlet_state[0].flow_vol.unfix()
+        #     m.fs.solex_rougher_strip.mscontactor.aqueous_inlet_state[0].flow_vol.unfix()
+        #     m.fs.solex_rougher_strip.mscontactor.organic_inlet_state[
+        #         0
+        #     ].conc_mass_comp.unfix()
+        #     m.fs.solex_rougher_strip.mscontactor.aqueous_inlet_state[
+        #         0
+        #     ].conc_mass_comp.unfix()
+        # elif unit == m.fs.solex_cleaner_load.mscontactor:
+        #     _log.info(f"Initializing {unit}")
+        #     # Fix feed states
+        #     m.fs.solex_cleaner_load.mscontactor.organic_inlet_state[0].flow_vol.fix()
+        #     m.fs.solex_cleaner_load.mscontactor.aqueous_inlet_state[0].flow_vol.fix()
+        #     m.fs.solex_cleaner_load.mscontactor.organic_inlet_state[
+        #         0
+        #     ].conc_mass_comp.fix()
+        #     m.fs.solex_cleaner_load.mscontactor.aqueous_inlet_state[
+        #         0
+        #     ].conc_mass_comp.fix()
+        #     # Re-solve unit
+        #     solver = SolverFactory("ipopt")
+        #     solver.solve(m.fs.solex_cleaner_load, tee=True)
+        #     # Unfix feed states
+        #     m.fs.solex_cleaner_load.mscontactor.organic_inlet_state[0].flow_vol.unfix()
+        #     m.fs.solex_cleaner_load.mscontactor.aqueous_inlet_state[0].flow_vol.unfix()
+        #     m.fs.solex_cleaner_load.mscontactor.organic_inlet_state[
+        #         0
+        #     ].conc_mass_comp.unfix()
+        #     m.fs.solex_cleaner_load.mscontactor.aqueous_inlet_state[
+        #         0
+        #     ].conc_mass_comp.unfix()
+        # elif unit == m.fs.solex_cleaner_strip.mscontactor:
+        #     _log.info(f"Initializing {unit}")
+        #     # Fix feed states
+        #     m.fs.solex_cleaner_strip.mscontactor.organic_inlet_state[0].flow_vol.fix()
+        #     m.fs.solex_cleaner_strip.mscontactor.aqueous_inlet_state[0].flow_vol.fix()
+        #     m.fs.solex_cleaner_strip.mscontactor.organic_inlet_state[
+        #         0
+        #     ].conc_mass_comp.fix()
+        #     m.fs.solex_cleaner_strip.mscontactor.aqueous_inlet_state[
+        #         0
+        #     ].conc_mass_comp.fix()
+        #     # Re-solve unit
+        #     solver = SolverFactory("ipopt")
+        #     solver.solve(m.fs.solex_cleaner_strip, tee=True)
+        #     # Unfix feed states
+        #     m.fs.solex_cleaner_strip.mscontactor.organic_inlet_state[0].flow_vol.unfix()
+        #     m.fs.solex_cleaner_strip.mscontactor.aqueous_inlet_state[0].flow_vol.unfix()
+        #     m.fs.solex_cleaner_strip.mscontactor.organic_inlet_state[
+        #         0
+        #     ].conc_mass_comp.unfix()
+        #     m.fs.solex_cleaner_strip.mscontactor.aqueous_inlet_state[
+        #         0
+        #     ].conc_mass_comp.unfix()
```

Comment:
This section is no longer needed since the SXInitializer is working, so you can just delete it